front-end/src/app/shared/layout/header/header.component.html

```diff
@@ -15,33 +15,38 @@
           <a class="nav-link" (click)="sidebar_hide()" [routerLink]="['/candidate']"
             routerLinkActive="item-active">CANDIDATES LIST <span class="sr-only">(current)</span></a>
         </li>
-        <li class="nav-item">
-          <a class="nav-link" (click)="logout()" [routerLink]="''">LOGOUT</a>
-        </li>
       </ul>
     </nav>
   </div>
   <!-- /#sidebar-wrapper -->
 
   <!-- Page Content -->
-  <div id="page-content-wrapper">
-    <nav id="id-nav" class="navbar navbar-expand">
-      <a class="navbar-brand" (click)="toggle()" id="menu-toggle">
-        <button class="btn">
-          <i class="fas fa-bars"></i>
-        </button>
-      </a>
-      <ul class="navbar-nav ml-auto">
+  <div id="page-content-wrapper" >
+    <nav id="id-nav" class="navbar navbar-expand" >
+      
+        <a class="navbar-brand" (click)="toggle()" id="menu-toggle">
+          <button class="btn">
+            <i class="fas fa-bars"></i>
+          </button>
+        </a>
+      
+      <ul class="navbar-nav ml-auto" (click)="toggleUserOpen()" >
         <li class="nav-item">
-          <button class="nav-link btn p-0" [routerLink]="['/user/data', user?.idUser]">
+          <a class="nav-link btn p-0" (clickOutside)="toggleUserClose($event)">
```

Comment:
Could you please add a shade or something to the profile photo when the user hover it with the mouse to indicate that it can be "clickable"  

![image](https://user-images.githubusercontent.com/23131424/77773070-7c4dec00-701f-11ea-85fd-f6bcae01c01c.png)